extensions/wikia/Rail/RailController.class.php

```diff
@@ -77,15 +81,15 @@ protected function getLazyRail() {
 
 		krsort( $railModules );
 
-		// TODO XW-2760 remove after experiment is done
-		$isAdMixExperimentEnabled = $this->request->getBool( 'isAdMixExperimentEnabled', false );
+		// TODO ADEN-5481 remove after experiment is done
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)